cluster-up/cluster/kind/common.sh

```diff
@@ -94,7 +94,7 @@ function _run_registry() {
         docker rm $REGISTRY_NAME || true
         sleep 5
     done
-    docker run -d --network=${network} -p $HOST_PORT:5000  --restart=always --name $REGISTRY_NAME registry:2
+    docker run -d --network=${network} -p $HOST_PORT:5000  --restart=always --name $REGISTRY_NAME quay.io/libpod/registry:2.7
```

Comment:
I suggest to use version 2.6, it has higher security score:
https://quay.io/repository/libpod/registry?tab=tags

Reply:
using the same version we are using on VM based providers, i think it is better to have the same version